force-app/main/default/lwc/tableWithAccounts/tableWithAccounts.js

```diff
@@ -1,57 +1,58 @@
 import { LightningElement, track } from 'lwc';
 import getDetails from '@salesforce/apex/AccountIntegrationController.getDetails';
+import patchDetails from '@salesforce/apex/AccountIntegrationController.patchDetails';
 import returnAccounts from '@salesforce/apex/AccountIntegrationController.returnAccounts';
+import updateReturnAccounts from '@salesforce/apex/AccountIntegrationController.updateReturnAccounts';
 import { ShowToastEvent } from 'lightning/platformShowToastEvent';
+import { refreshApex } from '@salesforce/apex';
 
 const pageSize = 7;   
 const columns = [
+    //add colum org
     {label: 'Name', fieldName : 'name'},
     {label: 'Account Number', fieldName : 'accountNumber', editable : true},
     {label: 'Ticker Symbol', fieldName : 'tickerSymbol', editable : true},
     {label: 'Rating', fieldName : 'rating', editable : true},
-    {label: 'Type', fieldName : 'type', editable : true}
+    {label: 'Type', fieldName : 'type', editable : true},
+    {label: 'Org', fieldName : 'org'}
 ];
-
 export default class TableWithAccounts extends LightningElement {
 
     @track listAcc = [];
     @track isLoading = false;
```

Comment:
not needed to track primitives, they are tracked by default. You need to track objects, arrays only